stdlib/public/runtime/Errors.cpp

```diff
@@ -471,3 +471,32 @@ void swift::swift_abortDisabledUnicodeSupport() {
                     "Unicode normalization data is disabled on this platform");
 
 }
+
+#if defined(_WIN32)
+// On Windows, exceptions may be swallowed in some cases and the
+// process may not terminate as expected on crashes. For example,
+// illegal instructions used by llvm.trap. Disable the exception
+// swallowing so that the error handling works as expected.
+__attribute__((constructor)) static void disableWinExceptionSwallowing() {
```

Comment:
Please use the reserved spelling:

```suggestion
__attribute__((__constructor__))
static void disableWinExceptionSwallowing() {
```

I wonder if we should make this portable and use the same technique that we use in the registrar to ensure execution at the appropriate time.

I wonder if we can rename this something like `ConfigureExceptionPolicy`?

Reply:
Can you expand on "I wonder if we should make this portable and use the same technique that we use in the registrar to ensure execution at the appropriate time." ? What's the registrar?

Reply:
https://github.com/apple/swift/blob/main/stdlib/public/runtime/SwiftRT-COFF.cpp#L100-L102

The use of constructor sections is how Windows does the constructor execution.

The registrar is the `swiftrt.obj` which registers the image with the runtime.